rollup/app/lwc/rollupOrderBy/rollupOrderBy.html

```diff
@@ -0,0 +1,73 @@
+<template>
```

Comment:
Just curious, did you consider making the HTML & JS for this `rollupOrderBy` LWC be a part of the existing lwc `rollupForceRecalculation` (instead of a separate LWC)? I haven't actually done it yet, but a few weeks ago, I was looking into how to [use multiple HTML & JS files within a single LWC](https://www.levelupsalesforce.com/multiple-html-files-in-a-component-bundle-lwc) - it doesn't seem like you currently have a use for this LWC outside of `rollupForceRecalculation`, but I'm not sure if you definitely could consolidate the 2 LWCs into 1.

Reply:
I could have consolidated them, yeah. I wanted to encapsulate the ordering logic, though; if I was more disciplined, I would have been more thorough while working on the full recalc app at organizing the distinct responsibilities of the component into various sub components. At the same time, since then I’ve tried to maintain a balance between encapsulation and over-architecting this side of the repo. The balance so far has been “same component since these responsibilities are similar” but here I chose to deviate from that since the “responsibilities” were tied to a different base SObject.